docs/databags.md

```diff
@@ -93,7 +93,10 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `['bag']` - A string that points to an externalized Apps Hash in which all keys (except this one) are valid. 
 * `[app]` - The name of an app to manage (disk name)
```

Comment:
The expected structure of the tarball is indeed that there is a top level directory that is the app name. (Since this databag is used to configure all apps, including those generated by chef, I need to know the name of the tarball). The example app in vagrant_repo/apps/my_app is built in this way by [in the Vagrantfile above](https://github.com/acharlieh/cerner_splunk/blob/a44768b5e83fabfe91ba8c0859698fab79b9f73b/Vagrantfile#L158)

If the tarball does not contain the named app, (as would be the case when the names don't match), the chef run [fails in the splunk_app resource execution, below](https://github.com/acharlieh/cerner_splunk/blob/a44768b5e83fabfe91ba8c0859698fab79b9f73b/libraries/splunk_app.rb#L208)

Another note. If there are multiple top level directories in a particular tarball, we only install the one that we want to install (per the configuration).